src/msg.c

```diff
@@ -213,7 +213,10 @@ _processKeyValue(int line, natsMsg *msg, char *endPtr, char **pPtr, char **lastK
     start = ptr;
     if (*ptr == '\r')
     {
-        if ((++ptr == endPtr) || ((*ptr == '\n') && (++ptr == endPtr)))
+        if ((++ptr == endPtr) ||
+            ((*ptr == '\n') && (++ptr == endPtr)) ||
+            ((*ptr == '\r') && (++ptr == endPtr)) ||
```

Comment:
I am not sure this is correct. We enter this "if" statement with `*ptr == '\r'`, so the check here would mean that we are ok with `\r\r`, which I don't think we should.

Reply:
Right, I'll re-group these